lib/context-util.c

```diff
@@ -139,18 +139,18 @@ tcti_tabrmd_init (void)
 
     rc = tss2_tcti_tabrmd_init(NULL, &size);
     if (rc != TSS2_RC_SUCCESS) {
-        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32, rc);
+        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32 "\n", rc);
```

Comment:
The only point I see for these is when printing hex dumps, or things with multiple invocations of printf. Perhaps we could format the data as a string and output it so we don't need to modify these.